README.md

```diff
@@ -35,17 +40,12 @@ can use to build your app as follows:
 pack build <app-name> -p <path-to-app> -b <path/to/cpython.cnb> -b <path/to/pip.cnb> -b <path/to/poetry.cnb> -b <path/to/poetry-install.cnb> -b build/buildpackage.cnb
 ```
 
-## Application Detection
-This buildpack detects on the presence of **exactly one** poetry script defined in `pyproject.toml`.
-More specifically, the buildpack will detect if `pyproject.toml` looks like the following:
-
-```
-[tool.poetry.scripts]
-some-script = "some.module:some_method"
-```
-
 ### Configuration
 
+#### Custom run command
```

Comment:
I think we should document that this feature also works for commands as well as scripts defined in the `pyproject.toml`. Maybe an example using the `python` command?

Reply:
@joshuatcasey what do you think to this idea of documenting an example with a command (e.g. `python`)

Reply:
@robdimsdale There is a sort of example in the section:

> Example: If `BP_POETRY_RUN_TARGET=default_app.server:run`, this buildpack will set a start command of `poetry run default_app.server:run`.

I could add a new example take from the [poetry docs](https://python-poetry.org/docs/cli/#run): `poetry run python -V`